Pod/Classes/TMPodspec.m

```diff
@@ -61,21 +76,125 @@ - (NSString *)version {
     return [self stringForKey:PodspecKeyVersion];
 }
 
-- (NSString *)summary {
-    return [self stringForKey:PodspecKeySummary];
+- (NSArray *)authors {
+    NSString *authorName = [self stringForKey:PodspecKeyAuthor];
+    NSArray *authorNames = [self arrayForKey:PodspecKeyAuthors];
+    NSDictionary *authorNamesAndAddresses = [self dictionaryForKey:PodspecKeyAuthors];
```

Comment:
That’d increase the nesting if I do it the way I’m thinking about, but yeah, we could. Which part of them being here bugs you: performance concerns, too many things in scope, something else?